plugins/@grouparoo/iterable/README.md

```diff
@@ -0,0 +1,5 @@
+# @Grouparoo/Iterable
+
+## Attribution
+
+The Iterable icon used in this plugin is from http://getlogo.net/iterable-inc-logo-vector-svg
```

Comment:
I think maybe these are usually square. Can you find one of those. `png` is fine.